.eslintrc.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  root: true,
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:ember-suave/recommended',
+  ],
+  env: {
+    qunit: true,
+    node: true,
+  },
+  globals: {
+    // 'expectAssertion': true,
+    // 'expectDeprecation': true,
+    // 'expectNoDeprecation': true,
+    // 'expectWarning': true,
+    // 'expectNoWarning': true,
+    // 'ignoreAssertion': true,
+    // 'ignoreDeprecation': true,
+
+    // Electron
+    'requireNode': true,
+    'processNode': true,
+
+    // A safe subset of "browser:true":
+    // 'window': true,
+    // 'document': true,
+    // 'setTimeout': true,
+    // 'clearTimeout': true,
+    // 'setInterval': true,
+    // 'clearInterval': true,
+
+    // 'Symbol': true,
+    // 'WeakMap': true,
+  },
+  rules: {
+    'comma-dangle': ['error', 'always-multiline'],
+    'ember-suave/no-const-outside-module-scope': 'off',
+    'ember-suave/no-direct-property-access': 'off',
+    'ember-suave/require-access-in-comments': 'off',
+    'newline-before-return': 'error',
+    'new-cap': 'off',
```

Comment:
Yeah, totally -- definitely with code style updates do fast & iterative changes to minimize merge conflicts :)

All of the others seem fine to me, and I think a number of Ember folks are backing off of the Ember destructuring practice (`no-direct-property-access`), although I still like it a lot. Agree that `require-access-in-comments` doesn't make sense for a project that doesn't expose a Javascript API. I have no strong feelings about `newline-before-return`.

`new-cap` is the one I was questioning -- maybe electron has a bunch of capital letter functions (I don't think I've personally run into them), but I haven't seen much code in *this* project that would violate the `new-cap` rule. Am I missing something -- does it actually cause more than just one or two errors?